enhancements/network/service-egress-traffic-steering.md

```diff
@@ -159,38 +169,50 @@ the egress traffic flow for the pod `10.244.1.6` on `ovn-worker2` towards an ext
 ```
 As mentioned earlier, for the opposite direction (ingress/external client initiates) to work properly the LoadBalancer provider needs to announce the service only from `ovn-worker`.
 
+When the `snat` field is not specified, `ovnkube-master` does not need to create any logical router policies because the egress packets of each pod would exit through the pod's node but will set the status field of the resource with `host: ALL` as decribed in [Network without SNAT](#Network-without-SNAT).
+
 #### Network
-In addition, an `EgressService` supports a `network` field.
-When it is specified the relevant `ovnkube-node` takes care of creating an additional ip rule and iptables rules on its host.
-Assuming an `EgressService` has `Network: blue`, a ClusterIP of 10.96.135.5 and its endpoints are 10.244.0.3 and 10.244.1.6, a `fwmark` will be internally assigned to the "blue" table and the following will be created on the host:
+The `EgressService` supports a `network` field to specify to which network the egress traffic of the service should be steered to.
+When it is specified the relevant `ovnkube-nodes` take care of creating ip rules on their host.
```

Comment:
Can you elaborate which `ovnkube-nodes` are relevant here?

Reply:
done
```
either the node which matches `Status.Host` or all of the nodes when `Status.Host` is "ALL"
```